### PR TITLE
bump expiry date for Alerting

### PR DESCRIPTION
**Pull request by soupdragon99**
Propose extending the guidance for six months, the guidance also needs updating.

### CODE REVIEW

**Comment by danielroseman** [comment]
I've pushed some changes which address the main issues I had with this page.

File: source/standards/alerting.html.md.erb [hunk 1]
```diff
@@ -61,14 +63,15 @@ track alerts, escalate them (automatically or manually), acknowledge them
...
 
-The default tool at GDS to be alerted of problems with your service
-is [PagerDuty](https://www.pagerduty.com/).
+Your monitoring system should use [PagerDuty](https://www.pagerduty.com/)
+to send alerts about problems with your service.
 
-We recommend using [Pingdom](https://www.pingdom.com/) as well
-in case PagerDuty becomes unavailable.
+We recommend using [Pingdom](https://www.pingdom.com/) as well, as a 
+top-level check on your service's availability.
```
Comment (soupdragon99) [inline source/standards/alerting.html.md.erb]: what do we mean by a top-level check?

**Comment by soupdragon99** [comment]
I think this guidance will need a full review, but let's get the date extended and this tweak in. 
@danielroseman could you approve this?

File: source/standards/alerting.html.md.erb (hunk 1 above)
Comment (danielroseman) [reply source/standards/alerting.html.md.erb]: Can't think of a better way to phrase this - the point is that all Pingdom does is request a page on your website to check it's actually up, it doesn't do any detailed monitoring.

**Pull request merged**